### PR TITLE
cache and synchronize assessment data

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: src/js/controllers/AssessmentController.js [hunk 1]
```diff
@@ -53,6 +56,17 @@
...
+        id: uuid(),
+        type: assessmentDaysCache.KEY,
+        date: $window.moment().toISOString().slice(0, 10),
+        assessmentDate: configuration.getCurrentAssessmentDate().toISOString().slice(0, 10),
+        isComplete: true,
+        weekNumber: configuration.getWeekNumber()
+      });
+    };
+
     this.showContent = function() {
```
Comment (michaelwehrley) [inline src/js/controllers/AssessmentController.js]: Do you want to return either `false`, `true`, or `truthy`?

Comment (ericcf) [inline src/js/controllers/AssessmentController.js]: No, just true or false.

**Comment by ericcf** [comment]
I'm going to go ahead and merge this. If you have any more comments, please let me know and I'll address them. Thanks!

File: src/js/controllers/AssessmentController.js [hunk 2]
```diff
@@ -161,6 +203,16 @@
       return this.model && this.model.isComplete();
     };
 
+    this.persistAction = function(event) {
+      participantActionsCache.persist({
+        id: uuid(),
+        type: participantActionsCache.KEY,
+        emittedAt: $window.moment().toISOString(),
+        elementLabel: angular.element(event.target).text().trim(),
```
Comment (michaelwehrley) [inline src/js/controllers/AssessmentController.js]: Glad to see you taking advantage of angular's jqLite.

**Pull request merged**